frontend/assets/stylesheets/objects/_grid.scss

```diff
@@ -113,10 +124,55 @@
     }
 }
 
-/* 3up Stacked (3 > 1)
+/* 3up Stacked (Cols per row: 3 on larger viewport > 1 on smaller)
    ========================================================================== */
 
+// 3 on desktop > 1 on tablet > 1 on mobile
 .grid--3up-stacked {
+    &.grid--equal-heights {
```

Comment:
Does this need to be a chained class? Can is just be an extra modifier we add as `grid--equal-heights`. If it *does* need to be chained can we reduce some duplication either by combining the classes or using a mixin, whichever makes more sense for you.